src/commands/start.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace StellarWP\Slic;
+
+if ( $is_help ) {
+	$help = <<< HELP
+	SUMMARY:
+		Starts containers in the stack.
+
+	USAGE:
+
+		<yellow>{$cli_name} {$subcommand} [service]</yellow>
+
+	EXAMPLES:
+
+		<light_cyan>{$cli_name} {$subcommand}</light_cyan>
+		Start all containers in the stack.
+
+		<light_cyan>{$cli_name} {$subcommand} wordpress</light_cyan>
+		Start the wordpress container in the stack.
+	HELP;
+
+	echo colorize( $help );
+	return;
+}
+
+$service = args( [ 'service' ], $args( '...' ), 0 )( 'service' );
+
+if ( ! $service ) {
+	start_all_services();
+	echo colorize( PHP_EOL . "✅ <green>All containers are running.</green>" . PHP_EOL );
```

Comment:
I'm not sure this will survive a non-TTY output, like that found in CI, nicely.
Nothing against prettier output, but it's something to keep an eye on.

Reply:
Hrm - that's a good thought. Let's see how it works out and then change it if the result is crummy. 💃